recipes/rbpbench/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "RBPBench" %}
-{% set version = "0.9" %}
-{% set sha256 = "247b9e0cabe5e4bd454653dbc1bfd0337b381b1e8d595bdd6d409b5693a57bda" %}
+{% set version = "1.0" %}
```

Comment:
Should the 'max_pin' below be changed to "x" ?

Reply:
Good question. I think I cannot 100% guarantee that I will NOT modify some of the existing parameters in the v1.x versions :) So I'd rather keep x.x.

Reply:
Great thx!